### PR TITLE
Add a config option to specify custom command-line option

### LINKED ISSUE #476: How can I append custom commandline option?
I'd like to add some custom commandline option. Is there any configuration for that in `fps_config.json`?

### PR DESCRIPTION
**Pull request by littlewu2508**
Closes: https://github.com/34736384/genshin-fps-unlock/issues/476

### CODE REVIEW

**Comment by littlewu2508** [comment]
I have compiled it on Debian with the modification of https://github.com/34736384/genshin-fps-unlock/pull/478: 
[unlockfps_nc_custom_option.zip](https://github.com/user-attachments/files/17430678/unlockfps_nc.zip)

File: unlockfps_nc/Model/Config.cs [hunk 1]
```diff
@@ -26,6 +26,7 @@ public class Config
         public int CustomResY { get; set; } = 1080;
         public int MonitorNum { get; set; } = 1;
         public int Priority { get; set; } = 3;
+        public string CustomOption { get; set; } = "";
```
Comment (34736384) [inline unlockfps_nc/Model/Config.cs]:
```suggestion
        public string AdditionalCommandLine { get; set; } = "";
```
'CustomOption' seems to be ambiguous about what it does

File: unlockfps_nc/Service/ProcessService.cs [hunk 2]
```diff
@@ -234,6 +234,7 @@ private string BuildCommandLine()
                 commandLine += "use_mobile_platform -is_cloud 1 -platform_type CLOUD_THIRD_PARTY_MOBILE ";
 
             commandLine += $"-monitor {_config.MonitorNum} ";
+            commandLine += $"{_config.CustomOption} ";
```
Comment (34736384) [inline unlockfps_nc/Service/ProcessService.cs]:
```suggestion
            commandLine += $"{_config.AdditionalCommandLine} ";
```

**Review by 34736384 — CHANGES_REQUESTED** [review]
Hello, please see the requested changes below

**Pull request merged**